src/rdo.rs

```diff
@@ -590,32 +592,36 @@ pub fn rdo_partition_decision(
         if subsize == BlockSize::BLOCK_INVALID {
           continue;
         }
+        pmv = best_pred_modes[0].mv;
+        if pmv.row != 0 || pmv.col != 0 {
+          let _b = 0;
```

Comment:
?

Reply:
:) AWCY is working hard to answer us... I ran for -s 1 and -s 3. Will share the results soon.
Btw, thx for spot my debug code to remove!